Seravo/seravo-plugin
# Add Geo IP login restrictions back

#### What are the main changes in this PR?

Adds back the possibility to restrict login to specific whitelisted countries only. Also adds a temporary classmap autoloader. 

WP-CLI commands to configure GEO restrictions:
- wp seravo geologin list
- wp seravo geologin allow <country> [--network]
- wp seravo geologin disallow <country> [--network]

Example usage:
1. No whitelist is configured by default:
```
$ wp seravo geologin list 
Restricted login: Disabled
```
2. Whitelist Finland:
```
$ wp seravo geologin allow fi --network
Success: Login from 'Finland' is now allowed on all blogs.
```
3. Login is now only allowed from Finland network wide:
```
$ wp seravo geologin list 
Restricted login: Enabled

Allow login on blog 'Joosuakoskinen' from:


Allow login network-wide (all blogs) from:

    - Finland (FI)
```



#### Manual testing steps?
This has already been reviewed once. It should be enough that basic functionality is tested:
- Login successfully before configuring any restrictions.
- Whitelist a random country and see that login fails.
- Whitelist `FI` and see that login works.
- Empty the whitelist and see that login works.
